src/groups/mqb/mqbnet/mqbnet_tcpsessionfactory.cpp

```diff
@@ -1599,7 +1599,7 @@ TCPSessionFactory::PortManager::addChannelContext(bmqst::StatContext* parent,
         bmqst::StatContextConfiguration portConfig(
             static_cast<bsls::Types::Int64>(port),
             &localAllocator);
-        bsl::shared_ptr<bmqst::StatContext> portStatContext =
+        bslma::ManagedPtr<bmqst::StatContext> portStatContext =
```

Comment:
Is this change required for cpp03?
Changing `shared_ptr` to `ManagedPtr` looks like a step back in this case.

Somehow we are able to build a `shared_ptr` later when we construct a `PortContext`

```
        struct PortContext {
            bsl::shared_ptr<bmqst::StatContext> d_portContext;
            bsl::size_t                         d_numChannels;
        };
```